src/multirust

```diff
@@ -1482,30 +1601,46 @@ cmd="${1-}"
 check_metadata_version
 
 case "$cmd" in
-    default)
+    default | override )
 	if [ -z "${2-}" ]; then
-	    err 'unspecified default toolchain. try `multirust help default`'
+	    err "unspecified $cmd toolchain. try \`multirust help $cmd\`"
 	fi
-	install_toolchain_if_not_installed "$2"
-	set_default "$2"
-	;;
 
-    override)
-	if [ -z "${2-}" ]; then
-	    err 'unspecified override toolchain. try `multirust help override`'
+	if [ "${3-}" = "--installer" -a -n "${4-}" ]; then
+	    update_custom_toolchain_from_installers "$2" "$4"
+	elif [ "${3-}" = "--installer" ]; then
+	    err 'unspecified installer. try `multirust help update`'
+	elif [ "${3-}" = "--copy-local" -a -n "${4-}" ]; then
+	    update_custom_toolchain_from_dir "$2" "${4-}" false
+	elif [ "${3-}" = "--link-local" -a -n "${4-}" ]; then
+	    update_custom_toolchain_from_dir "$2" "${4-}" true
+	elif [ "${3-}" = "--copy-local" -o "${3-}" = "--link-local" ]; then
+	    err "unspecified toolchain path. try \`multirust help $cmd\`"
+	elif [ -n "${3-}" ]; then
+	    err "unrecognized option ${3-}"
+	else
+	    install_toolchain_if_not_installed "$2"
 	fi
-	install_toolchain_if_not_installed "$2"
-	set_override "$2"
+	"$(set_$cmd "$2")"
 	;;
 
     update)
```

Comment:
it seems to me that `update` now is pretty much the same as `override` and `default` except for the `update_toolchain "$2"` call. I think it would be worth merging them to avoid duplicating the cli arguments parsing code.